pywr/parameters/_parameters.pyx

```diff
@@ -2006,5 +2006,32 @@ def load_parameter_values(model, data, values_key='values', url_key='url',
                          "Please provide either a '{}', '{}' or '{}' entry.".format(values_key, url_key, table_key, name=name, ptype=ptype))
     return values
 
+cdef class DiscountFactorParameter(Parameter):
```

Comment:
Can you move this above line 1909 (i.e. above the functions and below the last parameter class)?